bindings/python/algorithm/expose-model.cpp

```diff
@@ -83,6 +83,22 @@ namespace pinocchio
       return bp::make_tuple(ancestor_id, index_ancestor_in_support1, index_ancestor_in_support2);
     }
 
+    template<typename Scalar, int Options, template<typename, int> class JointCollectionTpl>
+    ModelTpl<Scalar, Options, JointCollectionTpl> transformJointIntoMimic(
```

Comment:
You can use the same pattern as findCommonAncestor_proxy, this avoid to specify the full signature in the definition.